app-tasks/rf/src/rf/cli.py

```diff
@@ -18,7 +18,7 @@
 logger = logging.getLogger('rf')
 
 
-if os.getenv('AWS_BATCH_JOB_ATTEMPT', '') == '3':
+if int(os.getenv('AWS_BATCH_JOB_ATTEMPT', '-1')) > 3:
```

Comment:
`>=` ?

Reply:
nope, email sends on the third failure. catching it here would cause it to exit before failing and sending the email on the third attempt

Reply:
Given that max retries is three though, this will never execute -- which is fine, given the original reason for this branch
was intermittent weird stall-out on ingests that shouldn't happen anymore, but if we're never going to get into this branch it should just be removed

Reply:
good point. I'll just take it out